pyproject.toml

```diff
@@ -13,6 +13,7 @@ gunicorn = "^20.1.0"
 psycopg2-binary = "^2.8.6"
 sentry-sdk = "^1.3.1"
 drf-spectacular = "^0.26.0"
+freezegun = "^1.2.2"
```

Comment:
This should be a dev.dependencies, it's only for testing not for production :thinking: